src/site/includes/google-analytics.liquid

```diff
@@ -0,0 +1,47 @@
+{% assign analytics_environment = buildtype %}
+{% if analytics_environment == 'localhost' %}
+  {% assign analytics_environment = 'vagovdev' %}
+{% endif %}
+
+{% case analytics_environment %}
+
+  {% when 'vagovdev' %}
+    {% capture google_analytics_js %}
+      {% include "src/site/assets/js/google-analytics/vagovdev.js" %}
+    {% endcapture %}
+
+    {% capture google_analytics_noscript %}
+      <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WFJWBD&gtm_auth=inC4EKQce9vlWpRVcowiyQ&gtm_preview=env-423&gtm_cookies_win=x"
+    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+    {% endcapture %}
+
+  {% when 'vagovstaging' %}
+    {% capture google_analytics_js %}
+      {% include "src/site/assets/js/google-analytics/vagovstaging.js" %}
+    {% endcapture %}
+
+    {% capture google_analytics_noscript %}
+      <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WFJWBD&gtm_auth=inC4EKQce9vlWpRVcowiyQ&gtm_preview=env-661&gtm_cookies_win=x"
+  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+    {% endcapture %}
+
+  {% else %}
+    {% capture google_analytics_js %}
+      {% include "src/site/assets/js/google-analytics/vagovprod.js" %}
+    {% endcapture %}
+
+    {% capture google_analytics_noscript %}
+      <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WFJWBD"
+  height="0" width="0" class="csp-inline-patch-footer"></iframe></noscript>
+    {% endcapture %}
+
+{% endcase %}
+
+<script data-e2e="analytics-script">
+  {{ google_analytics_js }}
+  recordEvent({ environment: '{{ buildtype }}' });
+  if ( self !== top ) {
+    recordEvent({ event: 'iframed', embeddedIn: document.referrer });
+  }
+</script>
+
```

Comment:
Analytics code is centralized to here. This template outputs a script element and exports a new variable, `google_analytics_noscript`, to be added into the top of the DOM.